examples/keycloak/README.md

```diff
@@ -0,0 +1,113 @@
+# Keycloak Setup Guide
+
+In order to use the Tenzir Platform with a Keycloak instance
+as Identity Provider, it is necessary to do some additional
+manual configuration.
+
+This document is intended to guide you through this process.
+
+We assume that you already have a Keycloak instance up
+and running that is reachable with a browser, for example
+by running the `keycloak` service from the docker compose file
+in this example. The same configuration steps apply when using
+an external instance.
+
+## Setting up the Keycloak Instance
+
+### Admin Login
+
+Navigate to the keycloak instance and log in as a user with admin permissions.
```

Comment:
```suggestion
Navigate to the Keycloak instance and log in as a user with admin permissions.
```

Reply:
I won't comment it everywhere, but please consistently spell Keycloak and Docker Compose correctly.